containerbuddy/logging.go

```diff
@@ -0,0 +1,92 @@
+package containerbuddy
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// LogConfig configures the log levels
+type LogConfig struct {
+	Level  string `json:"level"`
+	Format string `json:"format,omitempty"`
+	Output string `json:"output,omitempty"`
+}
+
+var defaultLog = &LogConfig{
+	Level:  "INFO",
+	Format: "go",
+	Output: "stdout",
+}
+
+func init() {
```

Comment:
So this makes sure we have some kind of sane logging to stdout even if the config file doesn't parse? Nice.